jobs/cloud_controller_clock/templates/cloud_controller_ng.yml.erb

```diff
@@ -59,6 +59,9 @@ jobs:
   diego_sync:
     timeout_in_seconds: <%= timeout %>
   <% end %>
+  <% if_p("cc.jobs.priorities") do |priorities| %>
+  properties: <%= priorities %>
```

Comment:
Should be `priorities:`, not `properties:`.

Reply:
And the `priorities` hash needs to be rendered as json (i.e. `priorities.to_json`).